### ISSUE TITLE
Showing name of sent phishingmail

### ISSUE THREAD

**Issue by stracire** [gophish/gophish#769]
Not an issue, but more a feature request. Sometimes I get confused which phishingmail I have sent. Is it possible to add the name of the sent e-mail to the statistic page?

Before I did use the duplicate function to view this, but I noticed that this function isn't displaying always the right sent phishingmail and landingspage.

**Comment by jordan-wright** [comment]
It sounds like what you're looking for is available on the campaign listing page if you hover over the campaign status. You might check out #552 for examples of what this looks like.

Let me know if this helps!

**Comment by stracire** [comment]
It does show some information. But the 'E-mail Template information' did not made it to the final build. There obvious are reasons for this decision so I fix this issue by just typing the e-mail template name into the campaign title.

**Comment by S0larflare** [comment]
Yeah the template name was removed from the api call Jordan was using when he slimmed down things a bit and I didn't see the value in adding a new api call just to get the template name. It is useful though so maybe it is worth modifying the api call/moving to a different one which has template name info in it too? I haven't looked at it in a good 6 months so can't remember if the call is used in other places (it likely is) so don't know whether quickly adding the extra info in would cause a performance impact or not.

**Comment by jordan-wright** [comment]
I'd be fine adding back in the template name. That's not something that is too heavyweight. I'll see if I can get that knocked out after finishing up some other current issues.
